pkg/registry/cluster/strategy.go

```diff
@@ -73,6 +74,10 @@ func (Strategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
 // PrepareForCreate is invoked on create before validation to normalize the object.
 func (Strategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
 	cluster := obj.(*clusterapis.Cluster)
+	cluster.Status = clusterapis.ClusterStatus{}
```

Comment:
I think this line is useless, why we initialize` cluster.Status` here? It is not a pointer.

Reply:
Just like k8s native resource like this:

https://github.com/kubernetes/kubernetes/blob/f461527491effcdbe23814b7481f46723cddda9e/pkg/registry/apps/deployment/strategy.go#L79

https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/apps/daemonset/strategy.go#L76

etc.

Reply:
I think the initialization status here can filters the situation where the status is specified during creation.

I feel that the changes in the status section can be done in another pr, so I will remove this change.